### PR TITLE
Removed spaces from `ci-none` info strings

### PR DESCRIPTION
**Pull request by WeeknightMVP**
Mitigates #179.

### CODE REVIEW

File: labs/Language/Basics.md [hunk 1]
```diff
@@ -1624,7 +1624,7 @@ correct by printing `Q.E.D.`. This means Cryptol has proven that your
 `R` is correct for all possible inputs (which is either `2^^96` for
 the 32-bit proof or `2^^192` for the 64-bit proof).
 
-```Xcryptol-session ci-none
+```Xcryptol-session-ci-none
```
Comment (weaversa) [inline labs/Language/Basics.md:1627]: Does this not work in the CI? The proofs seem pretty easy for Z3.

File: labs/SimonSpeck/SimonSpeck.md [hunk 2]
```diff
@@ -375,7 +375,7 @@ able to load the `SpeckTestVectors` module also located in the `Speck`
 folder as follows and verify that you have correctly implemented the
 functions:
 
-```Xcryptol-session ci-none
+```Xcryptol-session-ci-none
```
Comment (weaversa) [inline labs/SimonSpeck/SimonSpeck.md:378]: Same here. The tests pass pretty quickly in the comment. Does the CI need to ignore this?

Comment (WeeknightMVP) [reply labs/SimonSpeck/SimonSpeck.md:378]: I'll remove them and see how CI does.

Comment (WeeknightMVP) [reply labs/SimonSpeck/SimonSpeck.md]: Some of the proofs stall for different Cryptol or solver versions.

File: labs/Language/Basics.md [hunk 3]
```diff
@@ -2038,9 +2038,11 @@ lazyAbsMin x y = if x == 0 then 0 else min (abs x) (abs y)
 Does not produce an error when `x` is zero, regardless of the value of
 `y`. For instance:
 
-```Xcryptol-session
+```Xcryptol-session-ci-none
 labs::Language::Basics> lazyAbsMin 1 (0/0)
+```
 
+```Xcryptol-session
```
Comment (weaversa) [inline labs/Language/Basics.md:2045]: I don't think that's in the right spot...perhaps down one line?

File: labs/SimonSpeck/SimonSpeck.md (hunk 2 above)
Comment (WeeknightMVP) [reply labs/SimonSpeck/SimonSpeck.md]: That escalated quickly.

File: labs/Language/Basics.md (hunk 3 above)
Comment (WeeknightMVP) [reply labs/Language/Basics.md:2045]: There are a few parts of the form "Pretend you edit this file and then reload it; here's what happens".  I'm refactoring the ones I see and running the script locally until it passes.

File: labs/SimonSpeck/SimonSpeck.md (hunk 2 above)
Comment (WeeknightMVP) [reply labs/SimonSpeck/SimonSpeck.md]: We should decide whether we want exercises in `Basics.md`.  If so, we should do the same thing we do for most other exercises: fill them in in `BasicsAnswers.md` and verify our answers.

**Review by weaversa — APPROVED** [review]
Thanks @WeeknightMVP !

**Pull request merged**